src/newcharacter.cpp

```diff
@@ -158,6 +156,11 @@ struct points_left {
     }
 };
 
+std::string empty_line( size_t len )
```

Comment:
Seriously? Why?

Reply:
There was an empty line constant. So an empty line function fills the empty void.

Reply:
Wraping std::string( num, ' ' ) in empty_line( num ) isn't an improvement, it's just as verbose and adds no significant clarity to the code.